packages/annotations/src/add.ts

```diff
@@ -45,14 +45,9 @@ export interface IAddAnnotationsRequestOptions
 export function addAnnotations(
   requestOptions: IAddAnnotationsRequestOptions
 ): Promise<IAddFeaturesResult> {
-  const session = requestOptions.authentication as UserSession;
-  const author = session ? session.username : null;
-
   requestOptions.adds.forEach(function(anno) {
     const defaults = {
-      created_at: new Date().getTime(),
       status: "pending",
-      author,
```

Comment:
I assume the idea behind removing `updated_at` and `created_at` is that they are added on the server. 

However, why remove `author`? That seems like a breaking change.

Reply:
the server adds `author` as well. my guess is that it introspects it from the token.

Reply:
https://www.youtube.com/watch?v=3TUNSzbEypA